src/components/Viewport.ts

```diff
@@ -608,14 +608,11 @@ export default class Viewport {
 
   public updateBbox(): BoundingBox {
     const state = this.state;
+    const options = this.options;
+    const viewportElement = this.viewportElement;
+
     if (!state.cachedBbox) {
-      const bbox = this.viewportElement.getBoundingClientRect();
-      state.cachedBbox = {
-        x: bbox.left,
-        y: bbox.top,
-        width: bbox.width,
-        height: bbox.height,
-      };
+      state.cachedBbox = getBbox(viewportElement, options.useOffset, true);
```

Comment:
@WoodNeck 
I think x and y will look very different in the `calcNewVisiblePanelIndex` method. Is that okay?

Reply:
That looks not good, I think I should fix that.